backend/api_server/app.py

```diff
@@ -23,6 +25,39 @@
 configure_logging(APP_NAME)
 
 
+def has_datadog_env_vars():
```

Comment:
nit: this could be inlined with a comment.

Reply:
I created a function here because we might want to add more conditions. For example `gunicorn` integration may require some other type of checking. We might want to check deployment environment var as well. I could rename the function to something like `should_configure_datadog_tracing()`. Would that work?

Reply:
If that is how it will be used in the future then the proposed name makes sense.